src/components/scenes/CreateWalletSelectCryptoScene.tsx

```diff
@@ -93,11 +93,6 @@ const CreateWalletSelectCryptoComponent = (props: Props) => {
   })
 
   const handleNextPress = useHandler(async () => {
-    if (numSelected === 0) {
```

Comment:
As far as I can tell the only way to get this error is if the user has typed into the search bar and they press the done button on the keyboard without first selecting anything. The keyboard doesn't drop in that case (at least on android) so some kind of feedback for the user is preferable compared to nothing happening at all.

Reply:
The fade animation isn't enough, you can still click it while fading. 

As mentioned in the commit, we don't want to show errors to the user when they haven't done anything unexpected. Errors look really bad and should be reserved for unexpected things.